sw/device/silicon_creator/lib/error.h

```diff
@@ -152,7 +153,8 @@ enum module_ {
   X(kErrorRomExtBootFailed,           ERROR_(1, kModuleRomExt, kFailedPrecondition)), \
   \
   /* The high-byte of kErrorInterrupt is modified with the interrupt cause */ \
-  X(kErrorRomExtInterrupt,            ERROR_(0, kModuleRomExtInterrupt, kUnknown))
+  X(kErrorRomExtInterrupt,            ERROR_(0, kModuleRomExtInterrupt, kUnknown)), \
+  X(kErrorBootLogInvalid,             ERROR_(1, kModuleBootLog, kInternal))
```

Comment:
nit: empty line between different modules
```suggestion
  X(kErrorRomExtInterrupt,            ERROR_(0, kModuleRomExtInterrupt, kUnknown)), \
  \
  X(kErrorBootLogInvalid,             ERROR_(1, kModuleBootLog, kInternal))
```